handlers/aws/sqs_trigger.py

```diff
@@ -28,7 +29,7 @@ def _handle_sqs_continuation(
     sqs_records = lambda_event["Records"][current_sqs_record:]
     body = json.loads(sqs_records[0]["body"])
     body["Records"] = body["Records"][current_s3_record:]
-    body["Records"][0]["last_ending_offset"] = last_ending_offset + 1
+    body["Records"][0]["last_ending_offset"] = last_ending_offset
```

Comment:
in the initial design we had both the `ending_offset` (the end of the last line processed) and `beginning_offset` (the char at where the line starts in the file). the latest is now calculated by the iterator consumer doing a diff between the ending offset, the line length and eventual newline length (1 or 2 for linux or windows).
all the values were affected by bugs indeed, and rather than missing events we were starting the continuation from a `last_ending_offset ` that was not really the char at were we left, but before.
regarding the `+1`, it is not needed since the char to start from while fetching the file again is zero based